src/views/Alteration.vue

```diff
@@ -304,6 +307,18 @@ export default class Alteration extends Mixins(
       this.emitFetchError(err)
     }
 
+    // set current profile name to store for field pre population
```

Comment:
This should be in the main try block -- right after setFeePrices and before emitHaveData.